### DISCUSSION TITLE
Make separate commands for player options

### DISCUSSION THREAD

**Discussion by yngdrms** [Zrips/CMI#7335] [Enhancement]
A recent CMI update added a GUI with customization of various player options via /cmi options. Before that, various commands were available that allowed me to make a separate menu based on them that would be as customizable as possible.

I suggest adding each of the commands from /cmi options to alias.yml and making them optional so that everyone can enable/disable them as needed.

**Comment by Zrips** [comment]
This was done due to quite few commands doing similar things. It was compacted into one system and put under one command which can use GUI. You still have dedicated sub commands to perform same actions as before and if needed you can create your own custom alias with command like `/cmi aliaseditor new chatspy-cmi options chatspy toggle`

**Comment by mrfloris** [comment]
Here's my little drop in addon for cmi that maps old commands to the new ones, re-adding the legacy commands;

https://github.com/mrfdev/CMI/tree/master/Resources/Add-ons/legacytoggles

adjust to your liking.
